src/atom-environment.coffee

```diff
@@ -817,12 +817,15 @@ class AtomEnvironment extends Model
 
     @blobStore.save()
 
-  saveState: ->
+  saveState: (options) ->
```

Comment:
In general, I think the `isQuitting` property could be passed also to packages as well, so that they can make a more informed decision (as we do) on how to perform the serialization. What do you think? :thought_balloon: